students/slawinski_amadeusz/lesson_05_lists/comma_code.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+
+
+def nice_print(list):
+    if len(list) == 1:
```

Comment:
można dodać dodatkowy warunek dla pustej listy lub, gdy lista jest None:
`if not list:...`